js/ui/grid_core/ui.grid_core.filter_custom_operations.js

```diff
@@ -129,5 +127,7 @@ function noneOf(grid) {
     });
 }
 
-exports.anyOf = anyOf;
-exports.noneOf = noneOf;
+export {
+    anyOf,
```

Comment:
why is json exported instead of export two methods?

Reply:
It is the result of lebab conversion. I think it's ok